index.js

```diff
@@ -3,9 +3,11 @@
 module.exports = {
   "plugins": [
     "stylelint-scss",
-    "stylelint-order",
     "stylelint-selector-bem-pattern",
   ],
+  "extends": [
+    "stylelint-config-recess-order",
```

Comment:
The stylelint-order plugin is added inside the order config ([link](https://github.com/stormwarning/stylelint-config-recess-order/blob/main/index.js#L2)) so we don't need to add it on line 6